ui/src/reducers/index.js

```diff
@@ -40,10 +40,6 @@ const mxcubeReducer = combineReducers({
 });
 
 const rootReducer = (state, action) => {
```

Comment:
This wrapper function is no longer needed I think, you can inline the line below directly.